src/model/Shape.java

```diff
@@ -0,0 +1,28 @@
+package model;
+
+
+public abstract class Shape implements ShapesProperties {
+	
+	/************************************
```

Comment:
Because now in Java8, interface, abstract classes are alike.
It is in theory an interface, but as some code can be combined (color), it is better I think
For now, the color is simple, but after we can add gradient color, transparency...